### PR TITLE
blog: add "hosting by tlon", edit "state of urbit"

### PR DESCRIPTION
**Pull request by urcades**

### CODE REVIEW

**Comment by urcades** [comment]
This update includes "Hosting, by Tlon", which is a new post, and some link/header edits to "State of Urbit".

File: content/blog/20200930-hosting-by-tlon.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+We’ve talked for a while about how the ultimate destiny of Urbit is to escape the control of Tlon. For Urbit to really succeed, it has to be managed by its own community. We started Tlon in order to solidify the foundation of this young platform. Now that Urbit is starting to grow up, Tlon and Urbit will naturally drift apart a bit. 
+
+Urbit is meant to be universal, ubiquitous infrastructure. Tlon exists to make that infrastructure useful and usable. We expect Urbit, as a platform, to be cared for by the community in collaboration with the emerging [Urbit Foundation](josh’s blog post). We intend Tlon to become increasingly focussed on providing a great user experience for anyone interested in this new network, so the network can continue to grow and thrive.
```
Comment (matildepark) [inline content/blog/20200930-hosting-by-tlon.md]:
```suggestion
Urbit is meant to be universal, ubiquitous infrastructure. Tlon exists to make that infrastructure useful and usable. We expect Urbit, as a platform, to be cared for by the community in collaboration with the emerging [Urbit Foundation](https://urbit.org/blog/first-steps-towards-urbit-org/). We intend Tlon to become increasingly focussed on providing a great user experience for anyone interested in this new network, so the network can continue to grow and thrive.
```

File: content/blog/20200930-hosting-by-tlon.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The way we see it, hosting is the most important thing, next to Landscape, that Tlon can do to help Urbit continue toward widespread adoption. We don’t expect to be the only ones providing this service—and look forward to including more contributions from the community here on urbit.org as they continue to mature.
+
+If you’re curious to keep up with what Tlon is doing here, head to https://tlon.io and join the waiting list. We’ll be posting updates to that list as well as to the Tlon Transmissions notebook in the Urbit Community group. Going forward we’ll do our best to keep urbit.org neutral and focussed on the technology itself.
```
Comment (matildepark) [inline content/blog/20200930-hosting-by-tlon.md]:
```suggestion
If you’re curious to keep up with what Tlon is doing here, head to [tlon.io](https://tlon.io) and join the waiting list. We’ll be posting updates to that list as well as to the Tlon Transmissions notebook in the Urbit Community group. Going forward we’ll do our best to keep urbit.org neutral and focussed on the technology itself.
```

**Review by matildepark — APPROVED** [review]
fine otherwise

**Review by crrncllns — COMMENTED** [review]
New title for host post: Hosting the Future, author ~simfur-ritwed

File: content/blog/20201001-hosting-the-future.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+We’ve talked for a while about how the ultimate destiny of Urbit is to escape the control of Tlon. For Urbit to really succeed, it has to be managed by its own community. We started Tlon in order to solidify the foundation of this young platform. Now that Urbit is starting to grow up, Tlon and Urbit will naturally drift apart a bit. 
+
+Urbit is meant to be universal, ubiquitous infrastructure. Tlon exists to make that infrastructure useful and usable. We expect Urbit, as a platform, to be cared for by the community in collaboration with the emerging [Urbit Foundation](josh’s blog post). We intend Tlon to become increasingly focussed on providing a great user experience for anyone interested in this new network, so the network can continue to grow and thrive.
```
Comment (tylershuster) [inline content/blog/20201001-hosting-the-future.md:27]: Uh-oh

Comment (urcades) [reply content/blog/20201001-hosting-the-future.md:27]: yeah I'm fixing this now, it got messed up in a shuffle of files

**Pull request merged**